### DISCUSSION TITLE
Rewrite Host-relative URL's to be true-relative

### DISCUSSION THREAD

**Discussion by williewillus** [jekyll/jekyll#9448] [Q&A]
I have a Jekyll site. If I build it with `jekyll build`, pretty much everything from the CSS to the links all use Host-relative URL's `/foo.html` and `/bar.css`. Is there a way to make those true relative links based on the current page? Like if I were on `a/b/c.html`, the url's would be generated as `../../foo.html` and `../../bar.css` instead.

The reason is that I'm making a commemorative CD for an open source project of mine, and I want to build the website as it looked today, then ship it on the CD as raw HTML files that can be viewed by just double clicking index.html.

Is there a way to do that here?

**Comment by fauno** [comment]
You could use the [jekyll-relative-urls](https://rubygems.org/gems/jekyll-relative-urls) plugin :)
